sympy/core/basic.py

```diff
@@ -81,8 +83,8 @@ class Basic(Printable, metaclass=ManagedProperties):
                  '_assumptions'
                 )
 
-    _args: 'Tuple[Basic, ...]'
-    _mhash: 'Any'
+    _args: tuple[Basic, ...]
+    _mhash: Any
```

Comment:
Is it possible for `_mhash` to be more specific? It's supposed to be an `int` but it doesn't get initalised until it is needed and until then it is None. Should this be `Optional[int]` rather than `Any`?